### PR TITLE
#91 add order type requisition

### LINKED ISSUE #91: API: Order Type within a requisition
Currently requisitions do not contain orderType information.

What would be ideal:

```
{
    id: ...
    ...
    orderType: 'emergency'
}
```

For requisitions synced without an order type, maybe initialise to 'Standard' or some other term for a normal requisition?

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #91 and #120.

### CODE REVIEW

File: src/server/database/database.js [hunk 1]
```diff
@@ -211,29 +211,32 @@ export default class Database {
...
   async updateRequisition(requisition) {
     const {
       id,
-      storeId,
+      // storeId,
       supplierNameId,
       date,
       status,
-      masterlistId,
+      // masterlistId,
```
Comment (joshxg) [inline src/server/database/database.js]: are these comments intentional?

**Comment by joshxg** [comment]
Looks good. Let me know when you're ready to merge

File: src/server/database/database.js (hunk 1 above)
Comment (wlthomson) [reply src/server/database/database.js]: Yep, temporary hack to prevent the client editing read-only fields.

**Comment by wlthomson** [comment]
> Looks good. Let me know when you're ready to merge

Ready to merge.

**Pull request merged**